packages/identifier-resolution/src/types/common.ts

```diff
@@ -50,6 +50,10 @@ export function isCoseKeyIdentifier(identifier: ManagedIdentifierType): identifi
   return typeof identifier === 'object' && `kty` in identifier && ('baseIV' in identifier || 'x5chain' in identifier) && !('x5c' in identifier)
 }
 
+export function isOIDFEntityIdIdentifier(identifier: ManagedIdentifierType): identifier is ICoseKeyJson {
+  return typeof identifier === 'string' && identifier.startsWith('https://') 
```

Comment:
OIDFEntityIdIdentifier only supports https?

Reply:
I am not aware of other protocols. It's not like we'd be calling a non-SSL federation endpoint